packages/utils/i18n.ts

```diff
@@ -1,2 +1,2 @@
 export const isKorean = (text: string) =>
-  /([(\uAC00-\uD7AF)|(\u3130-\u318F)])+/gi.test(text)
+  /([\uAC00-\uD7AF|\u3130-\u318F])+/gi.test(text)
```

Comment:
```suggestion
  /[\uAC00-\uD7AF\u3130-\u318F]+/gi.test(text)
```

``` ts
/([\uAC00-\uD7AF|\u3130-\u318F])+/gi.test('(|)') // true
```

The `|`  is redundant

Reply:
IMO, keeping the `|` makes it more readable.

Reply:
![imPpP5.gif](https://i.328888.xyz/2023/04/10/imPpP5.gif)

Reply:
You are right, fixed it. Please take a look again.